relayer/src/chain/cosmos.rs

```diff
@@ -304,7 +304,7 @@ impl CosmosSdkChain {
         crate::telemetry!(query, self.id(), "query_consensus_params");
 
         Ok(self
-            .block_on(self.rpc_client.genesis())
+            .block_on(self.rpc_client.genesis::<serde_json::Value>())
```

Comment:
Doesn't look like this `query_consensus_params()` function is used anywhere. Should we remove it?

Reply:
Good catch!

Reply:
efd1950b1c10bf2d264e02203a95e2ebdff26cbe